src/layouts/GameOverModal/styles.css

```diff
@@ -0,0 +1,199 @@
+@font-face {
+  font-family: 'Star Jedi';
+  src: url('Starjedi.ttf') format('truetype');
+}
+
+.modal-container {
+  font-family: 'Star Jedi';
+  display: flex;
+  flex-direction: column;
+}
+
+.hidden {
+  display: none;
+}
+
+.modal-container__overlay {
+  background-color: black;
+  z-index: 1;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  opacity: 0.8;
+}
+
+.modal-container__game-over-modal {
+  width: 1000px;
+  max-width: 90%;
+  max-height: 100%;
+  background-color: white;
+  position: absolute;
+  top: 1rem;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  border-radius: 16px;
+  box-shadow: 0 0 20px rgba(255, 0, 0, 0.9);
+}
+
+.modal-container__close-button-container {
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: min-content;
+}
+
+.modal-container__header {
+  margin: 2rem auto;
+  font-size: 2rem;
+}
+
+.modal-container__summary {
+  max-width: 80%;
+  margin: 0 auto;
+  font-size: 1rem;
+  letter-spacing: 1px;
+  text-align: center;
+}
+
+.modal-container__main {
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-container__answers {
+  margin: 1rem auto;
+}
+
+.modal-container__yoda-image {
+  max-width: 200px;
+  margin: auto;
+}
+
+.modal-container__input-container {
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.modal-container__input-container input {
+  width: 80%;
+  height: 2.5rem;
+  line-height: 2rem;
+  font-size: 2rem;
+  border: 3px solid #ccc;
+  border-radius: 8px;
+  padding: 0.5rem 1rem;
+  z-index: 1;
+}
+
+.modal-container__input-description {
+  max-width: 80%;
+  text-align: center;
+  font-size: 1rem;
+  letter-spacing: 1px;
+  margin: 1rem 0;
+}
+
+.modal-container__button {
+  margin: 3rem auto;
+  width: 80%;
+  border-radius: 16px;
+  box-shadow: 0 0 20px rgba(255, 0, 0, 0.9);
+}
+
+@media only screen and (min-width: 360px) {
+}
+
+@media only screen and (min-width: 768px) {
+  .modal-container__game-over-modal {
+    position: absolute;
```

Comment:
You don't have to redeclare styles if they are the same as above. Styles concerning an element that are lowest in a file will be the ones a browser uses and this includes media queries.